solidity/deploy/43_authorize_light_relay_maintainer_proxy_in_light_relay.ts

```diff
@@ -0,0 +1,29 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const LightRelayMaintainerProxy = await deployments.get(
+    "LightRelayMaintainerProxy"
+  )
+
+  // TODO: Find who should call this step
```

Comment:
The deployer as it is now - see the other comment for this file.

Reply:
Removed the TODO in 53bcde8254bdbef80ededa96ff161d8ac99a88da and added ownership transfer of `LightRelay` from `deployer` to `governance` in 01f68f98587bd4c65179c151ead181932055ab86.